data/data/ovirt/affinity_group/main.tf

```diff
@@ -0,0 +1,18 @@
+resource "ovirt_affinity_group" "cluster_masters_affinity_group" {
+  count        = length(var.ovirt_control_plane_affinity_groups)
+  name         = var.ovirt_control_plane_affinity_groups[count.index]["name"]
+  cluster_id   = var.ovirt_cluster_id
+  priority     = var.ovirt_control_plane_affinity_groups[count.index]["priority"]
+  vm_positive  = false
+  vm_enforcing = false
+  vm_list      = var.masters_ids
+}
+
+resource "ovirt_affinity_group" "cluster_affinity_group" {
```

Comment:
Initially this affinity group should've contain all the machines in the cluster, but we decided to make some changes to the affinity group layout and now it will be just for the compute nodes so I will change it :)